docs/ref/templates/builtins.txt

```diff
@@ -684,9 +684,17 @@ This example includes the contents of the template whose name is contained in
 the variable ``template_name``::
 
     {% include template_name %}
+    
+The variable may also be any object with a ``render()`` method that accepts a 
+context. This allows you to reference a compiled ``Template`` in your context
 
-The variable may also be any object with a ``render()`` method that accepts a
-context. This allows you to reference a compiled ``Template`` in your context.
+Additionally, the variable may be an iterable of templates or objects with
+``render()`` methods, in which case the first that can be loaded will be used,
```

Comment:
That's not true, the variable cannot be an interable of objects with `render()` methods.